core/src/main/java/org/apache/calcite/sql/SqlSelectOperator.java

```diff
@@ -69,11 +70,12 @@ public SqlCall createCall(
         operands[3],
         (SqlNodeList) operands[4],
         operands[5],
-        (SqlNodeList) operands[6],
+        operands[6],
         (SqlNodeList) operands[7],
-        operands[8],
+        (SqlNodeList) operands[8],
         operands[9],
-        (SqlNodeList) operands[10]);
+        operands[10],
+        (SqlNodeList) operands[11]);
```

Comment:
It might be good to add a comment to each line mentioning which field each operand corresponds to (ex. that operands[0] corresponds to keywordList, etc).

Reply:
Sure I can add some comments here